App.js

```diff
@@ -14,13 +14,12 @@ ProfileRoute.queries = {
 };
 
 view App {
+  prop onLogout
+
   <Relay.RootContainer
-    {...{ route: new ProfileRoute() }}
+    route={new ProfileRoute()}
```

Comment:
Yeah, but I'd prefer to keep routing and use Relay. Relay routes are not url routes, so it should be possible to use Relay with Flint router. Thus the hack.